test/test_parser.py

```diff
@@ -17,8 +18,14 @@ class TestGetStream(unittest.TestCase):
     def test_getStream(self):
         """Test getStream() function, which performs an Adios advance operation.
         """
-        
-        prs = parser.Parser("test.cfg")
+
+        config = configparser.ConfigParser(interpolation=None)
+        config.read("test.cfg")
+
+        config['Adios']['Method'] = 'BP'
```

Comment:
I guess this is for quick testing... and also that's why removed the input from config file.

Reply:
Yes. In the test.cfg, it is set to use SST mode. That's why I manipulate the config file here. I will fix this part also to test SST mode. Or, maybe making separate unit test function for the SST mode.